app/views/users/index.html.haml

```diff
@@ -2,19 +2,24 @@
   %h1.entry-title= t('.title')
 .post-title-divider
   %span
+= search_form_for @q do | f |
+  = f.label  :email, 'Profile Email'
+  = f.search_field :email
+  / WIP
+  = f.submit
 .entry-content
   %table
     %thead
       %tr
-        %th= t('.email')
-        %th= t('.created')
-        %th= t('.last_login')
-        %th= t('.logged_in_count')
+        %th= sort_link(@q, :email, t('.email'))
+        %th= sort_link(@q, :created, t('.created'))
+        %th= sort_link(@q, :last_login, t('.last_login'))
+        %th= sort_link(@q, :logged_in_count, t('.logged_in_count'))
```

Comment:
Only fields that are included in the Ransack search form should use the `sort_link` helper.  That is, the fields have to be explicitly referenced in the generate SQL for the ransack sorting to work (internally it uses SQL "ORDER BY" to effect the sorting).